src/components/CourseTable/index.jsx

```diff
@@ -30,7 +30,7 @@ const CourseTable = () => {
 
   const formatCourseData = courses => courses.map(course => ({
     ...course,
-    title: (<Link to={`/courses/${course.uuid}`}>{course.title}</Link>),
+    title: (<Link to={`/courses/${course.uuid}/edit/`}>{course.title}</Link>),
```

Comment:
Something we should check in on as a team: Should the URL here just be /courses since we will not have a View Course?

Reply:
Happy for whatever. Let me know.

Reply:
We can postpone this change for a later date.